extras/pahma/app/components/blacklight/gallery/slideshow_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<div
```

Comment:
Overwriting this file to add aria attributes (formerly done via blacklight_gallery.js).